.ci/idea-format.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------
+# IntelliJ IDEA inspections for Checkstyle.
+#
+# Example Mac OS:
+# IDEA_PATH="/Applications/IntelliJ IDEA.app/Contents/MacOS/idea" ./.ci/idea-inspection.sh
+#
+# Example Linux:
+# export IDEA_PATH=$HOME/java/idea-IU-172.4574.11 && ./.ci/idea-inspection.sh
+# ----------------------------------------------------------------------
+
+PROJECT_DIR=$PWD
+FORMAT_PATH="$PROJECT_DIR/config/intellij-idea-code-style.xml"
+RESULTS_DIR="$PROJECT_DIR/target/format-results"
+
+if [[ -z $IDEA_PATH ]]; then
+    echo "IDEA_PATH variable not found."
+    # Try to search in path
+    IDEA_PATH="$(which idea)"
+    if [ -z "$IDEA_PATH" ]; then
+        echo "IntelliJ IDEA was not found in path."
+        exit -1
+    fi
+fi
+
+# Prepare results directory
+if [ -d "$RESULTS_DIR" ]; then
+    rm -rf "$RESULTS_DIR"
+fi
+mkdir -p "$RESULTS_DIR"
+
+# Compile the project with Maven
```

Comment:
```suggestion
:: Compile the project with Maven
```
whats difference between # and :: ? assuming its both comments?

https://stackoverflow.com/a/43158193